scoreboard/management/commands/wipe_db.py

```diff
@@ -1,30 +1,92 @@
 # Wipe everything in the database without changing the schema.
-from django.core.management.base import BaseCommand, CommandError
+from django.core.management.base import BaseCommand
 from django.db import transaction
 from scoreboard.models import *
 
+
+@transaction.atomic
+def wipe_leaderboard_fields(entity):
+    entity.longest_streak = 0
+    entity.unique_deaths = 0
+    entity.unique_ascs = 0
+    entity.unique_achievements = 0
+    entity.games_over_1000_turns = 0
+    entity.games_scummed = 0
+    entity.total_games = 0
+    entity.wins = 0
+    entity.trophies.clear()
+    entity.save()
+
+
+@transaction.atomic
+def clear_player_and_clan_fields():
+    for player in Player.objects.all():
+        wipe_leaderboard_fields(player)
+    for clan in Clan.objects.all():
+        wipe_leaderboard_fields(clan)
+
+
+@transaction.atomic
+def wipe_games():
+    Game.objects.all().delete()
+    clear_player_and_clan_fields()
+
+
+@transaction.atomic
+def wipe_all_but_clans():
+    Clan.objects.all().delete()
+    Player.objects.all().delete()
+    User.objects.all().delete()
+    clear_player_and_clan_fields()
+
+
+@transaction.atomic
+def wipe_non_fixtures():
+    Clan.objects.all().delete()
+    Player.objects.all().delete()
+    User.objects.all().delete()
+    Game.objects.all().delete()
+
+
+@transaction.atomic
+def wipe_all():
+    Clan.objects.all().delete()
+    Player.objects.all().delete()
+    User.objects.all().delete()
+    Game.objects.all().delete()
+    Achievement.objects.all().delete()
+    Conduct.objects.all().delete()
+    Trophy.objects.all().delete()
+    Source.objects.all().delete()
+
+
 class Command(BaseCommand):
-    help = 'Delete everything in the database'
-
-    # TODO: add options so that the default does not erase Clans or Players (but
-    # does zero Players' fields)
-    # something like:
-    #   0 args: wipes Games, leaves Players and Clans intact.
-    #   --full: wipes everything, including fixtures.
-    #   --all-but-clans: wipes everything except Clan (which can't easily be
-    #                    reconstructed).
-    #   --non-fixtures: wipes Player, Clan, Game and leaves the fixtures intact.
-    # Further TODO: print statements should reflect which models got wiped
+    help = 'Delete models in the database, with no args fixtures and player-clan relationships remain intact.'
+
+    def add_arguments(self, parser):
```

Comment:
Perhaps we should add another argument that is `--games` or similar -- behavior will be identical to the current behavior with no argument, but it makes it more explicit and user friendly when someone runs --help